pkgs/by-name/da/daggerfall-unity/package.nix

```diff
@@ -0,0 +1,132 @@
+{
+  alsa-lib,
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchurl,
+  fetchzip,
+  lib,
+  libGL,
+  libXScrnSaver,
+  libXcursor,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXxf86vm,
+  libpulseaudio,
+  libudev0-shim,
+  makeDesktopItem,
+  nix-update-script,
+  stdenv,
+  vulkan-loader,
+  pname ? "daggerfall-unity",
+  includeUnfree ? false,
+}:
+let
+  docFiles =
+    [
+      (fetchurl {
+        url = "https://www.dfworkshop.net/static_files/daggerfallunity/Daggerfall%20Unity%20Manual.pdf";
+        hash = "sha256-FywlD0K5b4vUWzyzANlF9575XTDLivbsym7F+qe0Dm8=";
+        name = "Daggerfall Unity Manual.pdf";
+        meta.license = lib.licenses.mit;
+      })
+    ]
+    ++ lib.optionals includeUnfree [
+      (fetchurl {
+        url = "https://cdn.bethsoft.com/bethsoft.com/manuals/Daggerfall/daggerfall-en.pdf";
+        hash = "sha256-24KSP/E7+KvSRTMDq63NVlVWTFZnQj1yya8wc36yrC0=";
+        meta.license = lib.licenses.unfree;
+      })
+    ];
+in
+stdenv.mkDerivation (finalAttrs: {
+  inherit pname;
+  version = "1.1.1";
+
+  src =
+    let
+      platformName =
+        if stdenv.hostPlatform.system == "x86_64-linux" then "linux_64bit" else "mac_universal";
```

Comment:
Do they offer platform specific binaries or just a universal one? Just know a lot of people prefer the single arch versions for reduced sizes.

Reply:
Nvm, just looked upstream and it doesn't look like it.

Reply:
The binaries are platform specific - see line 52.

Reply:
No, I meant for x86 vs aarch64